tests/flows/sync/test_agreement.py

```
@@ -496,6 +496,114 @@ def test_sync_agreement_prices_dry_run(
+        "PRD-1111-1111",
+        "USD",
+        None,
+    )
+    mock_mpt_update_agreement_subscription.assert_not_called()
+    mock_mpt_update_agreement.assert_not_called()
+    mock_get_template_data_by_adobe_subscription.assert_called_once()
+
+
+def test_sync_agreement_lines_missing_prices_airtable_and_mpt(
+    mocker,
+    mock_adobe_client,
```

Ruff (0.14.10)

Unused function argument: `mock_adobe_client`

(ARG001)